### PR TITLE
Skia build for mac os

### PR DESCRIPTION
**Pull request by velyan**
I was able to compile with Skia backend on Mac OS. This PR updates the `build.rs` script to support `macos`.

I've put the steps I followed to build Skia in a [gist](https://gist.github.com/velyan/7f474a09c51d2c7b658036913edfe003).

### CODE REVIEW

File: BUILD.adoc [hunk 1]
```diff
@@ -221,6 +221,23 @@ SKIA_DIR=path SKIA_LIB_DIR=path cargo build --release --features "skia-backend"
 
 == macOS
 
+We assume that you have already built Skia itself.
+
+Install `harfbuzz` using your distributive's package manager.
+
+Using BrewInstall you can install it via:
```
Comment (RazrFalcon) [inline BUILD.adoc]: >BrewInstall

Is it a typo?

File: BUILD.adoc [hunk 2]
```diff
@@ -221,6 +221,23 @@ SKIA_DIR=path SKIA_LIB_DIR=path cargo build --release --features "skia-backend"
 
 == macOS
 
+We assume that you have already built Skia itself.
+
+Install `harfbuzz` using your distributive's package manager.
```
Comment (RazrFalcon) [inline BUILD.adoc]: This line can be removed.

File: BUILD.adoc [hunk 3]
```diff
@@ -221,6 +221,23 @@ SKIA_DIR=path SKIA_LIB_DIR=path cargo build --release --features "skia-backend"
...
+
+Using BrewInstall you can install it via:
+
+```
+brew install harfbuzz
+```
+
+`SKIA_DIR` should point to a Skia directory that contains the Skia `include` directory.
+`SKIA_LIB_DIR` should point to a Skia directory that contains `libskia.so`.
```
Comment (RazrFalcon) [inline BUILD.adoc]: `libskia.so` ->  `libskia.dylib`

File: BUILD.adoc (hunk 1 above)
Comment (velyan) [reply BUILD.adoc]: fixed

File: BUILD.adoc (hunk 2 above)
Comment (velyan) [reply BUILD.adoc]: Updated

File: BUILD.adoc (hunk 3 above)
Comment (velyan) [reply BUILD.adoc]: fixed, thanks

Comment (RazrFalcon) [reply BUILD.adoc]: `dylib`, not `a`.

Comment (velyan) [reply BUILD.adoc]: ah I was building skia as a static library
okay fixed now, thanks!

**Comment by RazrFalcon** [comment]
Thanks!

**Pull request merged**